Add guide: Data grid with tiles

attitude opened the pull request:
Guide is intended for the v1.1 release

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/contember/docs/67)
<!-- Reviewable:end -->

VojtaStanek commented:
Hi!

I have a few notes about this. There are two ways to take form here:

a) Make it a **guide**: Make it complete for a particular use case from start to finish. In this case, it would be "Images" - starting from the schema (e.g. `Image`, `ImageUse` and using it in an article) and showing how to make a media gallery page and use the image picker. 

b) Make it a **reference**: You should provide examples of using the particular component (as you did here) and give an explanation of the props and behaviour. 


If I were you I would go with the reference since it's much easier (I wouldn't be so hard there in the code review - the barrier there is just not to lie). 

--- 

Just a side note: The component in the first step is replaceable by a `<Stack>` containing a `<Field />` component and `<FieldView />` component.

attitude commented:
@VojtaStanek, I have added reference to tile with `LiveCode` preview... tho to see some tiles in data-grid we need to fill Docs instance with data (I've already created issue to create Admin for Docs). I moved the guide to a separate WIP branch.

attitude commented:
Hi, this PR seems too outdated. Feel free to reopen if necessary.
